client/my-sites/themes/theme-showcase.scss

```diff
@@ -2,11 +2,83 @@
 @import "@wordpress/base-styles/mixins";
 
 body.is-section-themes-i4 {
-	--color-surface-backdrop: #fdfdfd;
+	&.theme-default.color-scheme {
+		--color-surface-backdrop: #fdfdfd;
+	}
 
-	&.theme-default .focus-content .layout__content {
+	.layout:not(.has-no-sidebar) .layout__content {
 		padding-top: var(--masterbar-height);
 	}
+
+	.theme__filters {
+		@include break-small {
+			padding: 24px 0;
+		}
+	}
+
+	.themes__selection .themes-list {
+		margin-top: 32px;
+	}
+
+	.upsell-nudge {
+		background-color: #f0f7fc;
+		border: none;
+		box-shadow: none;
+		margin-top: 32px;
+		margin-bottom: 0;
+		padding: 24px 30px;
+		border-radius: 4px;
+
+		&.is-dismissible {
+			.dismissible-card__close-button {
+				height: 16px;
+				width: 16px;
+
+				svg {
+					height: 16px;
+					width: 16px;
+				}
+			}
+		}
+
+		.banner__content {
+			gap: 16px;
+
+			@include breakpoint-deprecated( ">660px" ) {
+				gap: 32px;
+			}
+		}
+
+		.banner__title,
+		.banner__description {
+			color: var(--color-neutral-80);
+			font-size: $font-body-small;
+			line-height: 20px;
+		}
+
+		.banner__title {
+			font-weight: 500;
+		}
+
+		.banner__icon {
+			background-color: var(--studio-blue-50);
+			display: none;
+		}
+
+		.banner__icon-circle {
+			background-color: var(--studio-blue-50);
+		}
+
+		.banner__action .button {
+			background-color: var(--studio-blue-50);
+			border: none;
+			border-radius: 4px;
```

Comment:
Do we need this? The one in the /plugins page does not seem to have this

Reply:
The /plugins page uses this [file](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plugins/plugins-discovery-page/upgrade-nudge.jsx) for styling:
![Screen Shot 2022-11-29 at 10 55 55 AM](https://user-images.githubusercontent.com/797888/204427603-7fa99a94-7c10-460e-aa6d-8ed730e189df.png)

We've also added `border-radius: 4px` to make the button more aligned to the border-radius of the banner 🙂

Reply:
Ah sorry, I mean we have different styles for both pages now. Is this intended? Comparison:

| /themes | /plugins |
| -- | -- |
| Rounded<br><br><img width="180" alt="image" src="https://user-images.githubusercontent.com/1525580/204437071-cddc5c16-66c9-4586-8a1c-77aecbdf5ab5.png"> | Not rounded<br><br><img width="220" alt="image" src="https://user-images.githubusercontent.com/1525580/204437093-b58cebff-c766-4bb6-ac02-cf4be3b588e5.png">

Reply:
I would prefer to update the border-radius for the /plugins page as well. Thoughts @SaxonF?

Reply:
although I also prefer 4px, I think we should look at doing a system wide change at some point and stick with 2px for now as it looks like its default. I need to update Figma to reflect this as well.